draft-ietf-quic-transport.md

```diff
@@ -470,7 +472,9 @@ this version, it contains:
   * 01: 1-RTT packet (packet number size = 1)
   * 02: 1-RTT packet (packet number size = 2)
   * 03: 1-RTT packet (packet number size = 4)
+  * 07: 1-RTT packet (packet number size = 4, with packet number echo)
```

Comment:
Makes sense to me. Doesn't this mean we should undefine one-byte packet numbers everywhere, though? (That seems like a distinct PR)